### ISSUE TITLE
Unable to batch export srt subtitles both at once

### ISSUE THREAD

**Issue by i990049** [chidiwilliams/buzz#364]
And everytime I export srt files，it opens software such as subtitle edit .   How to set export subtitles without opening subtitle edit?

**Comment by Sircam19** [comment]
> And everytime I export srt files，it opens software such as subtitle edit . How to set export subtitles without opening subtitle edit?

That's a function of the settings (i.e. file associations) that you have chosen previously. 

Not sure from your question what operating system your using but assuming windows then this MIGHT point you in the right direction...Good luck. 

https://superuser.com/questions/904188/how-can-i-unassociate-a-file-type-from-a-program-without-a-registry-edit

**Comment by chidiwilliams** [comment]
Yeah, that works, but I can also add a setting to let you decide if you want to open the subtitle by default. Thanks.

**Comment by i990049** [comment]
> Yeah, that works, but I can also add a setting to let you decide if you want to open the subtitle by default. Thanks.

I just want to export all the transcribe srt files both at once without running any soft like subtitle edit. Now I have to repeat this one by one and close subtitle edit one by one.

**Comment by chidiwilliams** [comment]
> > Yeah, that works, but I can also add a setting to let you decide if you want to open the subtitle by default. Thanks.
> 
> I just want to export all the transcribe srt files both at once without running any soft like subtitle edit. Now I have to repeat this one by one and close subtitle edit one by one.

Got it. Will be fixed in the next release.
